### PR TITLE
Update toml library and add test

### PR DESCRIPTION
**Pull request by bcl**
This pull request includes:

- [X] adequate testing for the new functionality or fixed issue

- [ ] 1st commit of any `rhbz#` related PR contains bug reproducer; CI reports FAIL or
- [X] PR contains automated tests for new functionality and
- [ ] QE has approved reproducer/new tests and
- [ ] Subsequent commits provide bug fixes without modifying the reproducer; CI reports PASS and
- [ ] QE approves this PR; RHBZ status is set to `MODIFIED + Verified=Tested`

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -74,8 +74,8 @@ jobs:
         WORKDIR=$(readlink -f pkg/jobqueue/dbjobqueue/schemas)
         pushd $(mktemp -d)
         go mod init temp
-        go get -u github.com/jackc/tern
-        go run github.com/jackc/tern migrate -m "$WORKDIR"
+        go install github.com/jackc/tern@latest
```
Comment (teg) [inline .github/workflows/tests.yml:77]: Could we pin a specific version to avoid incompatible changes biting us in the future?

Comment (bcl) [reply .github/workflows/tests.yml:77]: The problem here isn't the version, it's that go now wants us to use install instead of get.

**Comment by bcl** [comment]
Closing, no chance of the PR working at this point.

**Comment by ondrejbudai** [comment]
@bcl I think that this bug was fixed on `main` by basically dependabot updating the dep. However, it would be nice to have the new test on `main` as well. Would it take much effort to "port" it?

**Comment by ondrejbudai** [comment]
Ignore me, this was done in https://github.com/osbuild/osbuild-composer/pull/3099, I failed at using the `git log`.

**Pull request closed without merging**